src/main/java/org/fundacionjala/coding/carlos/countfrequentitem/CountFrequentItem.java

```diff
@@ -0,0 +1,30 @@
+package org.fundacionjala.coding.carlos.countfrequentitem;
+
+public class CountFrequentItem {
+
+    /**
+     * @param arrayNumbers to find the most frequent item
+     * @return the number of times that the most frequent item is repeated in an array.
+     */
+    public int getMostFrequent(final int[] arrayNumbers) {
+        int number = 0;
+        int repeated = 0;
+        for (int arrayNumber : arrayNumbers) {
+            number = arrayNumber;
+            if (repeated < verifyFrequency(number, arrayNumbers)) {
```

Comment:
You are executing `verifyFrequency()` method two times, please consider store the result in a variable to reuse the value.
```java
int frequency = verifyFrequency(number, arrayNumbers);
if(repeated < frequency) {
  repeated = frecuency;
}
```